.changeset/shaggy-readers-approve.md

```diff
@@ -0,0 +1,5 @@
+---
+"@directus/app": patch
+---
+
+Updated the Editor.js version (used for Block Interface) to 2.29, introducing the Slash key as a shortcut for opening the toolbox and the Tab key to navigate through blocks
```

Comment:
Should we include improvements/features from updated dependencies in our changelog? 🤔

Reply:
This seemed relevant enough 🤔